sql/migrations/reports/000004_alter_reports_add_tr_tp_columns.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE reports
+ADD COLUMN IF NOT EXISTS transformation_id TEXT DEFAULT '',
+ADD COLUMN IF NOT EXISTS transformation_version_id TEXT DEFAULT '',
```

Comment:
@Jayachand Shouldn't this be `INT` like for tracking-plan ?

Reply:
transformation_version_id is a 32 character string whereas trackingplan version is a number